crmprtd/ec_swob/download.py

```diff
@@ -69,7 +70,7 @@ def get_url_list(
 
         # either return them or add to the queue
         for url in urls:
-            if search_pattern.search(url):
+            if search_pattern_dfo.search(url) or search_pattern.search(url):
```

Comment:
If you're able to get the combined pattern working, then this change should be able to be reverted.